test/unit/safeLogger.ts

```diff
@@ -1,7 +1,7 @@
 import * as chai from 'chai';
 import * as chaiAsPromised from 'chai-as-promised';
 
-import { logOrNothing } from '../../src';
+import { safeLogger } from '../../src';
```

Comment:
Actually safeLogger.ts (src & test) should be renamed to safe-logger.ts to follow our coding conventions.

Reply:
👍